### PR TITLE
Update fixture production

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/530
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/161
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/89
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1461

### CODE REVIEW

File: BackofficeBundle/DataFixtures/MongoDB/LoadNodeRouteDocumentData.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 /**
  * Class LoadNodeRouteDocumentData
  */
-class LoadNodeRouteDocumentData extends AbstractFixture implements OrderedFixtureInterface, OrchestraFunctionalFixturesInterface, ContainerAwareInterface, OrchestraProductionFixturesInterface
+class LoadNodeRouteDocumentData extends AbstractFixture implements OrderedFixtureInterface, OrchestraFunctionalFixturesInterface, ContainerAwareInterface
```
Comment (nicolasThal) [inline BackofficeBundle/DataFixtures/MongoDB/LoadNodeRouteDocumentData.php]: I would leave this class as production fixture

**Comment by itkg-alavieille** [comment]
@nicolasThal  pr updated

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**